po/ro.po

```diff
@@ -0,0 +1,703 @@
+msgid "key can not be empty"
+msgstr "cheia nu poate să fie empty"
+
+#: src/config.rs:791
+msgid "error: unknown option '{}' in section [bin]"
+msgstr "eroare: opțiune necunoscută '{}' în secțiunea [bin]"
+
+#: src/config.rs:891
+msgid "error: unknown option '{}' in section [options]"
+msgstr "erorare: opțiune necunoscută '{}' în secțiunea [options]"
+
+#: src/config.rs:896
+msgid "option '{}' does not take a value"
+msgstr "opțiunea '{}' nu necesită(primește) o valoare"
+
+#: src/config.rs:947 src/install.rs:1796
+msgid "There are {n} providers available for {pkg}:"
+msgstr "Există {n} provideri disponibili pentru {pkg}:"
+
+#: src/config.rs:961 src/info.rs:57 src/info.rs:96 src/install.rs:1755
+#: src/install.rs:1804
+msgid "Repository"
+msgstr "Repository"
+
+#: src/config.rs:983
+msgid "  failed to sync {}"
+msgstr " nu s-a putut sincroniza {}"
+
+#: src/devel.rs:129 src/download.rs:179
+msgid "Querying AUR..."
+msgstr "Se interoghează AUR..."
+
+#: src/devel.rs:142 src/devel.rs:166 src/install.rs:694
+msgid "failed to parse srcinfo for '{}'"
+msgstr "nu s-a putut procesa srcinfo pentru '{}'"
+
+#: src/devel.rs:184
+msgid "Looking for devel repos..."
+msgstr "Se caută devel repos..."
+
+#: src/devel.rs:193
+msgid "failed to save devel info"
+msgstr "nu s-au putut salva devel info"
+
+#: src/devel.rs:382
+msgid "package does not have an update"
+msgstr "pachetul nu are un update disponibil"
+
+#: src/devel.rs:417
+msgid "failed to lookup: {}"
+msgstr "nu s-a putut efectua căutarea: {}"
+
+#: src/devel.rs:445
+msgid "invalid json: {}"
+msgstr "json invalid: {}"
+
+#: src/download.rs:83
+msgid "packages not in the AUR: "
+msgstr "pachetele nu se găsesc în AUR: "
+
+#: src/download.rs:94
+msgid "marked out of date: "
+msgstr "marcat ca expirat: "
+
+#: src/download.rs:105
+msgid "orphans: "
+msgstr "orphans: "
+
+#: src/download.rs:206
+msgid "could not get current directory"
+msgstr "nu s-a putut obține directorul curent"
+
+#: src/download.rs:210
+msgid "can not get repo packages: asp is not installed"
+msgstr "nu s-au putut obține pachetele din repo: asp nu este instalat"
+
+#: src/download.rs:231
+msgid "Missing ABS packages "
+msgstr "Lipsă pachete ABS "
+
+#: src/download.rs:250 src/download.rs:276 src/download.rs:530
+#: src/download.rs:543 src/download.rs:557 src/exec.rs:72 src/exec.rs:113
+#: src/exec.rs:122 src/exec.rs:148 src/exec.rs:161 src/exec.rs:186
+#: src/exec.rs:200 src/exec.rs:221 src/exec.rs:234 src/install.rs:744
+#: src/install.rs:807 src/install.rs:845 src/keys.rs:74
+msgid "failed to run:"
+msgstr "nu s-a putut rula:"
+
+#: src/download.rs:263
+msgid "does not contain PKGBUILD: skipping"
+msgstr "nu conține PKGBUILD: se sare peste"
+
+#: src/download.rs:294
+msgid " ({total>padding$}/{total}) downloading: {pkg}"
+msgstr " ({total>padding$}/{total}) se descarcă: {pkg}"
+
+#: src/download.rs:317
+msgid "Downloading PKGBUILDs..."
+msgstr "Se descarcă PKBUILDs..."
+
+#: src/download.rs:321
+msgid " PKGBUILDs up to date"
```

Comment:
Missing space

Reply:
Saw the typo, fixed now, thanks for pointing it out.